lib/common-lib.sh

```diff
@@ -5,28 +5,27 @@
 
 IFS=$'\n'
 
+# If the repo contains a .formatting-directory file, only files in the specified directories will be returned (one directory per line).
+# If .formatting-directory doesn't exist then all files in the repository are checked.
+function directories_to_check() {
+    locations_to_diff=''
+	[ -e ".formatting-directory" ] && locations_to_diff=$( cat .formatting-directory | sed 's/ /\\ /g' );
```

Comment:
Please update the README with instructions about how to add multiple directories. Include a note that directories with spaces in the name should not be escaped with a `\`

(also the indentation here is off)